### ISSUE TITLE
[amp-sidebar][iOS] DOM ordering incorrect after closing sidebar

### ISSUE THREAD

**Issue by sparhami** [ampproject/amphtml#16400]
When closing a sidebar on iOS, the DOM order is different from when you opened the sidebar. This impacts screen readers.

For example, with `test/manual/amp-sidebar-toolbar.amp.html`, after closing the sidebar, the focus is on the open button, which is near the end of the DOM. You then need to swipe backwards to get to the content. If you never opened the sidebar, then the ordering is correct.

**Comment by aghassemi** [comment]
we move sidebar to iOS's only fixedlayer lazily first time in opens. The focus management (i.e. moving focus back to opener, and moving focus on sidebar on open) should be helping with a11y but in general fixedlayer breaks DOM order as its side-effect.

**Comment by jridgewell** [comment]
This could be fixed by the new iOS Shadow DOM scroll embed. It no longer reparents elements added to the fixed layer.

**Comment by aghassemi** [comment]
closing as all the iOS fixed element issues are on track to be removed when iOS fully launches the fixes Igalia has been doing.
